activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   Support `where` with custom operators (`>`, `>=`, `<`, and `<=`).
```

Comment:
I think we should call them `comparison operators` instead of `custom`. Ref: https://www.w3resource.com/sql/comparison-operators/sql-comparison-operators.php#:~:text=A%20comparison%20(or%20relational)%20operator,two%20NULL%20expressions%20returns%20UNKNOWN).

Reply:
👍